### ISSUE TITLE
Players in Velocity cannot receive payment notifications.

### ISSUE THREAD

**Issue by ArifErdoganK** [YiC200333/XConomy#124]
**Version of XConomy**  
Spigot XConomy V2.25.8
Velocity XConomy V2.25.8

**Server version**
Velocity 3.2.0 SNAPSHOT-296
Spigot 1.8.8

**Storage-type**  
MySQL

**Describe the issues**  
We have multiple servers connected to Velocity, and when players send money to players on different servers, those on different servers do not receive payment notifications.

---

**Comment by YiC200333** [comment]
Do you read FQA?
https://github.com/YiC200333/XConomy/blob/master/FAQ/English.md

**Comment by YiC200333** [comment]
Please confirm the mode used by Velocity

**Comment by ArifErdoganK** [comment]
I have read it, and everything seems to be set up correctly.

console.yml SyncData settings:
```
#SyncData settings
SyncData:
  #Whether to enable data synchronization between multiple servers
  enable: true
  #BungeeCord Redis
  #================================================== BungeeCord Channel ==================================================
  #After BungeeCord sync is turned on, the console and other plugins will not be able to modify the balance in the server without player
  #Please Set bungeecord to true in the spigot.yml
  #==================================================   Redis  Channel   ==================================================
  #Require 'slf4j' dependencies
  channel-type: Redis
  #Server ID, keep the ID and the MySQL settings is the same between the servers requiring synchronization
  sign: Faction

```
database.yml Redis settings:
```
#Redis settings
Redis:
  host: localhost
  port: 6379
  db-index: 1
  pool-settings:
    max-total: 10
    max-idle: 10
    min-idle: 0
  auth:
    user: ''
    pass: 'xxxxxx'
```


Console log:
```
[23:19:45] [INFO]: [XConomy] Enabling XConomy v2.25.8
[23:19:45] [INFO]: [XConomy] UUID-Mode: SemiOnline
[23:19:45] [INFO]: [XConomy] Language: ENGLISH
[23:19:45] [INFO]: [XConomy] Saving method - MySQL
[23:19:45] [INFO]: [XConomy] - Starting...
[23:19:45] [INFO]: [XConomy] - Driver does not support get/set network timeout for connections. (Receiver class com.mysql.jdbc.JDBC4Connection does not define or inherit an implementation of the resolved method 'abstract int getNetworkTimeout()' of interface java.sql.Connection.)
[23:19:45] [INFO]: [XConomy] - Added connection com.mysql.jdbc.JDBC4Connection@1c7b95c
[23:19:45] [INFO]: [XConomy] - Start completed.
[23:19:45] [INFO]: [XConomy] MySQL successfully connected
[23:19:45] [INFO]: [XConomy] Redis successfully connected
[23:19:45] [INFO]: [XConomy] Creating Redis listening thread
[23:19:45] [INFO]: [XConomy] XConomy successfully enabled
[23:19:45] [INFO]: [XConomy] Found PlaceholderAPI
[23:19:45] [INFO]: [XConomy] Subscribe redis channel success, channel XConomy_Sync
[23:19:45] [INFO]: [PlaceholderAPI] Successfully registered internal expansion: xconomy [2.25.8]
[23:19:45] [INFO]: [XConomy] PlaceholderAPI successfully hooked
[23:19:45] [INFO]: [XConomy] ===== YiC =====
```

**Comment by YiC200333** [comment]
This problem is fixed in 2.25.11 version
